.soliumrc.json

```diff
@@ -4,16 +4,28 @@
   "rules": {
     "imports-on-top": 0,
     "variable-declarations": 0,
-    "indentation": ["error", 4],
-    "quotes": [2, "double"],
     "no-empty-blocks": 0,
     "arg-overflow": 0,
     "uppercase": 0,
     "operator-whitespace": 0,
+    "array-declarations": "error",
+    "indentation": ["error", 4],
+    "quotes": ["error", "double"],
+    "conditionals-whitespace": "error",
+    "comma-whitespace": "error",
+    "semicolon-whitespace": "error",
+    "function-whitespace": "error",
+    "lbrace": "error",
+    "mixedcase": "warning",
+    "camelcase": "warning",
+    "no-unused-vars": "error",
+    "blank-lines": "warning",
+    "whitespace": "warning",
+    "pragma-on-top": "warning",
     "security/no-low-level-calls": 0,
     "security/no-inline-assembly": 0,
     "security/no-call-value": 0,
     "security/no-block-members": 0,
-    "security/enforce-explicit-visibility": 0
+    "security/enforce-explicit-visibility": "error"
```

Comment:
I probably should have been more clear (though I do think what you added is good):
These are the new ones I was referring to that I think we should add:
```
no-throw
no-sha3
no-tx-origin
```

Source: https://github.com/duaraghav8/solium-plugin-security#list-of-rules

Reply:
Right now they all default to enabled (I think?) but we should include them as explicitly enabled.

Reply:
Also, can we follow the `error`/`warning` syntax for everything instead of some being `0` and `1`?

Reply:
+1 on turning `0` to `"off"` but what do you mean by explicitly enabling?

Reply:
To clarify, there is no explicit enable other than specifying an error level.

Reply:
Reading again I think you're just re-iterating that we should be including these rules in our config. Gonna assume that's the case

Reply:
At the moment, `security/no-throw`, `security/no-sha3` and `security/no-tx-origin` don't show up anywhere in this configuration file, yet they are rules the linter has enabled because they are on-by-default.  I would like to add a line for each of those rules so they show up in our config as enabled as errors.  This is the same as you did for a bunch of other rules here, but for the security plugin.